Description: Fix some spelling errors
 Fix spelling errors in the code and in three man pages
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=3111
Last-Update: 2016-09-23

--- slurm-llnl-16.5.4.orig/doc/man/man1/salloc.1
+++ slurm-llnl-16.5.4/doc/man/man1/salloc.1
@@ -746,7 +746,7 @@ the next periodic accounting sample.
 
 .TP
 \fB\-\-mem\-per\-cpu\fR=<\fIMB\fR>
-Mimimum memory required per allocated CPU in MegaBytes.
+Minimum memory required per allocated CPU in MegaBytes.
 Default value is \fBDefMemPerCPU\fR and the maximum value is \fBMaxMemPerCPU\fR
 (see exception below). If configured, both of parameters can be
 seen using the \fBscontrol show config\fR command.
@@ -874,7 +874,7 @@
 Specify information pertaining to the switch or network.
 The interpretation of \fItype\fR is system dependent.
 This option is supported when running Slurm on a Cray natively.  It is
-used to request using Network Performace Counters.
+used to request using Network Performance Counters.
 Only one value per request is valid.
 All options are case in\-sensitive.
 In this configuration supported values include:
--- slurm-llnl-16.5.4.orig/doc/man/man1/sbatch.1
+++ slurm-llnl-16.5.4/doc/man/man1/sbatch.1
@@ -842,7 +842,7 @@ the next periodic accounting sample.
 
 .TP
 \fB\-\-mem\-per\-cpu\fR=<\fIMB\fR>
-Mimimum memory required per allocated CPU in MegaBytes.
+Minimum memory required per allocated CPU in MegaBytes.
 Default value is \fBDefMemPerCPU\fR and the maximum value is \fBMaxMemPerCPU\fR
 (see exception below). If configured, both parameters can be
 seen using the \fBscontrol show config\fR command.
@@ -970,7 +970,7 @@ that the \fB\-\-cpus\-per\-task\fR optio
 Specify information pertaining to the switch or network.
 The interpretation of \fItype\fR is system dependent.
 This option is supported when running Slurm on a Cray natively.  It is
-used to request using Network Performace Counters.
+used to request using Network Performance Counters.
 Only one value per request is valid.
 All options are case in\-sensitive.
 In this configuration supported values include:
--- slurm-llnl-16.5.4.orig/src/common/slurmdbd_defs.c
+++ slurm-llnl-16.5.4/src/common/slurmdbd_defs.c
@@ -980,9 +980,9 @@ extern slurmdbd_msg_type_t str_2_slurmdb
 		return DBD_STEP_START;
 	} else if (!xstrcasecmp(msg_type, "Get Jobs Conditional")) {
 		return DBD_GET_JOBS_COND;
-	} else if (!xstrcasecmp(msg_type, "Get Transations")) {
+	} else if (!xstrcasecmp(msg_type, "Get Transactions")) {
 		return DBD_GET_TXN;
-	} else if (!xstrcasecmp(msg_type, "Got Transations")) {
+	} else if (!xstrcasecmp(msg_type, "Got Transactions")) {
 		return DBD_GOT_TXN;
 	} else if (!xstrcasecmp(msg_type, "Add QOS")) {
 		return DBD_ADD_QOS;
@@ -1400,13 +1400,13 @@ extern char *slurmdbd_msg_type_2_str(slu
 		if (get_enum) {
 			return "DBD_GET_TXN";
 		} else
-			return "Get Transations";
+			return "Get Transactions";
 		break;
 	case DBD_GOT_TXN:
 		if (get_enum) {
 			return "DBD_GOT_TXN";
 		} else
-			return "Got Transations";
+			return "Got Transactions";
 		break;
 	case DBD_ADD_QOS:
 		if (get_enum) {
--- slurm-llnl-16.5.4.orig/src/sacct/options.c
+++ slurm-llnl-16.5.4/src/sacct/options.c
@@ -297,7 +297,7 @@ sacct [<OPTION>]
                  -p or -P options are not specified.\n\
      -D, --duplicates:                                                      \n\
 	           If SLURM job ids are reset, some job numbers will        \n\
-	           probably appear more than once refering to different jobs.\n\
+	           probably appear more than once referring to different jobs.\n\
 	           Without this option only the most recent jobs will be    \n\
                    displayed.                                               \n\
      -e, --helpformat:                                                      \n\
--- slurm-llnl-16.5.4.orig/doc/man/man1/srun.1
+++ slurm-llnl-16.5.4/doc/man/man1/srun.1
@@ -1310,7 +1310,7 @@ applies to job and step allocations.
 Specify information pertaining to the switch or network.
 The interpretation of \fItype\fR is system dependent.
 This option is supported when running Slurm on a Cray natively.  It is
-used to request using Network Performace Counters.
+used to request using Network Performance Counters.
 Only one value per request is valid.
 All options are case in\-sensitive.
 In this configuration supported values include:
